app/Actions/Database/StartPostgresql.php

```
@@ -315,19 +335,132 @@ private function add_custom_conf()
+            $configPath = PgBackrestService::CONFIG_PATH;
+            $archiveCommand = "test ! -f {$configPath}/pgbackrest.conf || (command -v pgbackrest >/dev/null 2>&1 && pgbackrest --stanza={$stanza} archive-push %p)";
+
+            if (! str($content)->contains('archive_mode')) {
+                $content .= "\narchive_mode = on";
+            }
+
+            if (str($content)->contains('archive_command')) {
+                $content = preg_replace("/archive_command\s*=\s*'[^']*'/", "archive_command = '{$archiveCommand}'", $content);
+            } else {
+                $content .= "\narchive_command = '{$archiveCommand}'";
+            }
```

PHPMD (2.15.0)

The method add_custom_conf uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)